api_spec.yaml

```diff
@@ -204,7 +214,7 @@ paths:
         '500':
           $ref: '#/components/responses/500'
```

Comment:
Merge the latest into this. Make sure that the older jwks and handshake endpoint is deprectaed